Docs 3436 scale 22.02 update disk article (#1399)
truenas/documentation: content/SCALE/SCALEUIReference/DataProtection/SMARTTestsSCALE.md

Next, select the test **Type**.
Test types differ based on the drive connection, ATA or SCSI:

{{< hint info >}}

> Note: We currently do not support manual S.M.A.R.T. tests on NVMe devices.

Reviewer: using the hint boxes removes the need to include Note: in the sentence, suggest the following:
```suggestion
Manual S.M.A.R.T. tests on NVMe devices is currently not supported.
```
